components/form/SelectFormControl.js

```diff
@@ -33,7 +33,7 @@ SelectFormControl.propTypes = {
   label: PropTypes.string,
   options: PropTypes.arrayOf(
     PropTypes.shape({
-      id: PropTypes.string.isRequired,
+      value: PropTypes.string.isRequired,
```

Comment:
I just got an explanation from @pydanny about this. He says `value` is option-specific whereas `id` is for the whole select component.